src/utils/time/index.ts

```diff
@@ -0,0 +1,11 @@
+export function getUNIXTime(dateTime: Date) {
+    if (!dateTime || isNaN(dateTime.getTime())) {
+        return null;
+    }
+    const unixTime = dateTime.getTime() * 1000;
+    if (unixTime <= 0) {
+        return null;
```

Comment:
iirc, unixTime will be negative for date < 1,1,1961

Reply:
This is intentional. we did it some time back when  we were getting a negative date Time value of a recent file and decided to ignore negative values all together

Reply:
cool.  By any chance do you have the file or remember why it was happening? Might as well add a reference to that PR here so that others don't get confused.   cc @vishnukvmd